packages/foam-vscode/src/features/tags-tree-view/index.ts

```diff
@@ -136,7 +124,6 @@ export class Tag extends vscode.TreeItem {
     this.description = `${this.notes.length} reference${
       this.notes.length !== 1 ? 's' : ''
     }`;
-    this.tooltip = getContainsTooltip(this.notes.map(n => n.title));
```

Comment:
this can actually be achieved with the new data structure in I think an elegant way using `resolveTreeItem` above.

Let me know if it makes sense

Reply:
Yes, the only thing I am doubting here is the performance of it. It basically means it will start to resolve the entire tree for this tooltip. Given the content of the tooltip, I don't know if that's worth the effort.

Reply:
> It basically means it will start to resolve the entire tree for this tooltip

not sure what you mean by "entire tree". The `resolveTreeItem` is meant to be lazily evaluated for a single tree item when hovered. See https://code.visualstudio.com/api/references/vscode-api#TreeDataProvider%3CT%3E

I agree the tooltip is not essential, so I am happy to go with your judgement as to whether to re-implement it lazily or not 👍